documentation/source/user_section/installation/linux.rst

```diff
@@ -4,6 +4,9 @@
 Installation for Linux
 **********************
 
+.. warning::
+
+  If you use Windows Subsystem for Linux (WSL), please refer to the :ref:`Windows installation section <installation/windows>`.
```

Comment:
Our doc build failed, and the warning is:

```
WARNING: undefined label: installation/windows
```

So it looks like Sphinx is trying to parse `installation/windows` as a label. To fix this, you could probably just change it to:

```suggestion
  If you use Windows Subsystem for Linux (WSL), please refer to the :ref:`Windows installation section <wsl-installation>`.
```

To match the reference used here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/edit/master/documentation/source/user_section/installation/windows.rst

Reply:
Ah, sorry! One other nitpick. There should be 3 spaces and not 2. (Sphinx is really finicky about spacing.

I'll edit the suggestion above.

Reply:
@joshuacwnewton Just added the space and used the reference.

I will setup a `rST` preview tool on my station to avoid any problem like that.

Reply:
Sphinx allows you to build the docs locally! You just need to install SCT with its `docs` dependencies. :)

https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Documentation%3A-Building-the-documentation#local-builds

Reply:
@joshuacwnewton Tested and approved. I will be able to work correctly on documentation for next PR ;)
Thanks for the link!